Source/Core/VideoBackends/D3D/D3DShader.cpp

```diff
@@ -54,10 +54,12 @@ bool CompileVertexShader(const std::string& code, D3DBlob** blob)
 		file << code;
 		file.close();
 
-		PanicAlert("Failed to compile vertex shader!\nThis usually happens when trying to use Dolphin with an outdated GPU or integrated GPU like the Intel GMA series.\n\nIf you're sure this is Dolphin's error anyway, post the contents of %s along with this error message at the forums.\n\nDebug info (%s):\n%s",
-						filename.c_str(),
-						D3D::VertexShaderVersionString(),
-						(const char*)errorBuffer->GetBufferPointer());
+		PanicAlertT("Failed to compile vertex shader!\n"
+		            "This usually happens when trying to use Dolphin with an outdated GPU or integrated GPU like the Intel GMA series.\n\n"
```

Comment:
The "This usually happens when trying to use Dolphin with an outdated GPU or integrated GPU" bit isn't really true anymore; pretty sure that problem mostly went away when we started requiring OpenGL 3.